lib/bamboo/test.ex

```diff
@@ -65,8 +67,44 @@ defmodule Bamboo.Test do
   @doc """
   Imports Bamboo.Test and Bamboo.Formatter.format_email_address/2
   """
+  defmacro __using__(process_name: process_name) do
+    quote do
+      setup tags do
+        if tags[:async] do
```

Comment:
Maybe I can do it with this: https://github.com/elixir-lang/elixir/blob/master/lib/ex_unit/test/ex_unit/assertions_test.exs#L591